src/EditorServicesCommandSuite/CodeGeneration/Refactors/CommandSplatRefactor.cs

```diff
@@ -32,32 +37,181 @@ internal CommandSplatRefactor(IRefactorUI ui)
             string variableName,
             CommandAst commandAst,
             bool newLineAfterHashtable,
+            bool allParameters,
+            bool mandatoryParameters,
+            bool noHints,
+            EngineIntrinsics executionContext,
             IRefactorUI ui = null)
         {
+            /*
+            High level design
+                1. Get parameters from Ast, resolve (positional, partial) parameters with StaticParameterBinder.
+                2. Resolve parameterset, [to be able to determine if a param is mandatory]
+                3. Gather relevant parameter data: name, value, isMandatory, type
+                    3a. (optional) add other parameters from CommandInfo + parameter set name. name, value, isMandatory, type
+                    3b. (optional) filter out non-mandatory params
+                4. Get longest parametername [to be able to format all '=' signs]
+                5. Sort // do we want a certain order?
+                6. Write parameters, values (if present), and (optional) type hint
+                    //  --> Parameter class moet iets van een ToWriteAssignment ofzo krijgen zie class Write, en class PowerShellScriptWriter.
+                    // naast 'internal void WriteHashtableEntry(string key, Action valueWriter)' moet er ook een komen met uitgelijnde '='.
+                7. Display parameter binding errors
+
+            */
```

Comment:
Agreed, I'll remove it. This was left in, because the commit holding the changes that reflect this oultine was quite a big refactor compared to previous work. I follow up on the numbered list troughout the GetEdits() function, that i'll clean up too.